### DISCUSSION TITLE
Nested selection under Group

### DISCUSSION THREAD

**Discussion by revanthmahesh** [fabricjs/fabric.js#7765] [Q&A] [answered]
expecting a table to be drawn on top of image, for now iam using lines and rectangle to active this as in [https://codesandbox.io/s/horse-remote-section-forked-5p6jrv?file=/src/index.js](https://codesandbox.io/s/horse-remote-section-forked-5p6jrv?file=/src/index.js). when i group them, i could not select the objects individually ie: lines to move along the axis. 

Not sure whether i can achieve this using fabric js.

**Comment by ShaMan123** [comment, marked as answer]
Luckily it's now possible
https://codesandbox.io/s/horse-remote-section-forked-809epu?file=/src/index.js

Still alpha so keep an eye over #7670
